lib/pages/company_site/eti/popups/groups_binding_popup.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module CompanySite
+  module ETI
+    class Table
+      class GroupsBindingPopup < self
+        div(:popup, css: '.groups-popup__title')
+        elements(:groups_tree, css: '.groups-popup__tree li')
+        button(:save, css: '.groups-popup__save-button')
+
+        def wait_for_visible
+          popup_element.when_visible
+        end
+
+        #TODO Надо бы пофиксить, чтобы с другими спеками было идентично
```

Comment:
Layout/LeadingCommentSpace: Missing space after #.<br>Style/CommentAnnotation: Annotation keywords like TODO should be all upper case, followed by a colon, and a space, then a note describing the problem.